io.github.CatalystDevOrg.Catalyst.json

```diff
@@ -0,0 +1,61 @@
+{
+  "app-id": "io.github.CatalystDevOrg.Catalyst",
+  "base": "org.electronjs.Electron2.BaseApp",
+  "base-version": "23.08",
+  "runtime": "org.freedesktop.Platform",
+  "runtime-version": "23.08",
+  "sdk": "org.freedesktop.Sdk",
+  "command": "run.sh",
+  "separate-locales": false,
+  "finish-args": [
+    "--share=ipc",
+    "--share=network",
+    "--socket=wayland",
+    "--socket=x11",
+    "--socket=pulseaudio",
+    "--device=dri",
+    "--talk-name=org.freedesktop.Notifications",
+    "--talk-name=org.kde.StatusNotifierWatcher",
+    "--talk-name=org.gtk.Notifications"
+  ],
+  "modules": [
+    {
+      "name": "catalyst",
+      "buildsystem": "simple",
+      "build-commands": [
+        "ar x catalyst_3.5.4_amd64.deb",
+        "tar -xf data.tar.zst",
+        "cp -r usr/* /app/",
+        "install -Dm644 icon-512x512.png /app/share/icons/hicolor/512x512/apps/io.github.CatalystDevOrg.Catalyst.png",
+        "mv /app/share/applications/catalyst.desktop /app/share/applications/io.github.CatalystDevOrg.Catalyst.desktop",
+        "install -Dm644 io.github.CatalystDevOrg.Catalyst.appdata.xml /app/share/metainfo/io.github.CatalystDevOrg.Catalyst.metainfo.xml",
+        "desktop-file-edit --set-key Exec --set-value \"/app/bin/run.sh\" ${FLATPAK_DEST}/share/applications/io.github.CatalystDevOrg.Catalyst.desktop",
+        "desktop-file-edit --set-icon ${FLATPAK_ID} ${FLATPAK_DEST}/share/applications/io.github.CatalystDevOrg.Catalyst.desktop",
+        "install run.sh /app/bin/"
+      ],
+      "sources": [
+        {
+          "type": "file",
+          "url": "https://raw.githubusercontent.com/CatalystDevOrg/Catalyst/1b99f30773bdfe9a0453dece38683059b0c55c6f/assets/icon-512x512.png",
+          "sha256": "2fba9fe630a0de3a9d04a8978158ad53edde42a61ed7bbe58be91bb65bca062a"
+        },
+        {
+          "type": "file",
+          "url": "https://github.com/CatalystDevOrg/Catalyst/releases/download/v3.5.4/catalyst_3.5.4_amd64.deb",
+          "sha256": "614be6176073a8f60a3fd6c24fad55e8ec70969314f07f590e39166669de2a65"
+        },
+        {
+          "type": "file",
+          "path": "io.github.CatalystDevOrg.Catalyst.appdata.xml"
+        },
+        {
+          "commands": [
+            "/app/bin/catalyst --no-sandbox"
```

Comment:
this should use zypak

https://github.com/refi64/zypak

(it's already in the base app)

Reply:
Fixed